LICENSE

```diff
@@ -1,4 +1,4 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+This work by Kyle R. Conway (https://kylerconway.com) is licensed under the Creative Commons Attribution-ShareAlike 4.0
```

Comment:
I think it's just different in the art world, but I've no worries. Do let me know if anything else is needed.